solcjs

```diff
@@ -14,12 +14,21 @@ var smtsolver = require('./smtsolver.js');
 process.removeAllListeners('uncaughtException');
 var commander = require('commander');
 
+const parseOptimizeRuns = function(value) {
+  const parsedValue = parseInt(value, 10);
+  if (isNaN(parsedValue)) {
+    return value;
+  }
+  return parsedValue;
+};
+
 var program = new commander.Command();
 program.name('solcjs');
 program.version(solc.version());
 program
   .option('--version', 'Show version and exit.')
   .option('--optimize', 'Enable bytecode optimizer.', false)
+  .option('--optimize-runs <optimize-runs>', 'The number of runs specifies roughly how often each opcode of the deployed code will be executed across the life-time of the contract.', parseOptimizeRuns)
```

Comment:
This line is pretty long. I'd wrap it.
I'd also add part of the description that `solc` uses:
```suggestion
  .option(
    '--optimize-runs <optimize-runs>',
    'The number of runs specifies roughly how often each opcode of the deployed code will be executed across the lifetime of the contract. ' +
    'Lower values will optimize more for initial deployment cost, higher values will optimize more for high-frequency usage.',
    parseOptimizeRuns
  )
```